### ISSUE TITLE
Custom dataset training

### ISSUE THREAD

**Issue by Xzy765039540** [huliangxiao/GaussianAvatar#6]
Very great project!!!
I used[ InstantAvatar's](https://github.com/tijiang13/InstantAvatar) preprocessing method to get data similar to People Snapshot Dataset with simple modifications can be trained in your project, but my trained characters are very unclear and hands can't be displayed. I would like to ask what causes this and when the custom data script will be released?
Regards.
![18001_gt](https://github.com/huliangxiao/GaussianAvatar/assets/91797762/df2e1479-bad8-440f-90a5-9d3028762e1c)
![18001_pred](https://github.com/huliangxiao/GaussianAvatar/assets/91797762/3b1423b0-370b-4002-ad85-7ae173326bfe)

**Comment by huliangxiao** [comment]
Hi, you need set pose_op_start_iter=10 (10 epochs for initial training) to let pose refinement start.

**Comment by Xzy765039540** [comment]
> Hi, you need set pose_op_start_iter=10 (10 epochs for initial training) to let pose refinement start.

Thanks for the reply.
After I tried to modify pose_op_start_iter=10 it is still training very fuzzy, I have compared the processed data and found that only smpl makes a difference. May I ask what method you are using to estimate smpl?
Kind regards
![08001_gt](https://github.com/huliangxiao/GaussianAvatar/assets/91797762/aa59b3e4-8e92-4ce2-8a62-8a4a8b700848)
![08001_pred](https://github.com/huliangxiao/GaussianAvatar/assets/91797762/03ea27b6-d709-4ab3-a5cd-42f790785d63)

**Comment by huliangxiao** [comment]
Just the same as you.
I directly employ the preprocessing scripts in InstantAvatar.
I will run the code again to find out whether i miss something or not.

**Comment by Xzy765039540** [comment]
> Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.

By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose

**Comment by huliangxiao** [comment]
> > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> 
> By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose

We use [ProxyCap](https://s based on [Przhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet.
I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).

**Comment by Xzy765039540** [comment]
> > > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> > 
> > 
> > By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose
> 
> We use [ProxyCap](https://zhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet. I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).

thanks,I will try it

**Comment by huliangxiao** [comment]
> > > > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> > > 
> > > 
> > > By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose
> > 
> > 
> > We use [ProxyCap](https://zhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet. I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).
> 
> thanks,I will try it

I know why the results are unclear, you need to set lpips_start_iter=10~30 (when to use lpips loss).
Sorry~

**Comment by Xzy765039540** [comment]
> > > > > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> > > > 
> > > > 
> > > > By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose
> > > 
> > > 
> > > We use [ProxyCap](https://zhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet. I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).
> > 
> > 
> > thanks,I will try it
> 
> I know why the results are unclear, you need to set lpips_start_iter=10-30 (when to use lpips loss). Sorry
Yes, I also found this problem Also I found that the pose obtained with the [ReFit ](https://github.com/yufu-wang/ReFit)project includes hand information I compared the pose from the People Snapshot Dataset and found that the matrices are exactly the same If you train with the pose obtained from ReFit you might get better results.

**Comment by huliangxiao** [comment]
> > > > > > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> > > > > 
> > > > > 
> > > > > By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose
> > > > 
> > > > 
> > > > We use [ProxyCap](https://zhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet. I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).
> > > 
> > > 
> > > thanks,I will try it
> > 
> > 
> > I know why the results are unclear, you need to set lpips_start_iter=10-30 (when to use lpips loss). Sorry
> > Yes, I also found this problem Also I found that the pose obtained with the [ReFit ](https://github.com/yufu-wang/ReFit)project includes hand information I compared the pose from the People Snapshot Dataset and found that the matrices are exactly the same If you train with the pose obtained from ReFit you might get better results.

Thanks!

**Comment by ZhenyanSun** [comment]
> > > > > > Just the same as you. I directly employ the preprocessing scripts in InstantAvatar. I will run the code again to find out whether i miss something or not.
> > > > > 
> > > > > 
> > > > > By comparing the result of instantavatar preprocessing and your demo data, I found that instantavatar uses romp to estimate the smpl. romp will not have hand pose in the prediction result (the author fills it with zeros), but your demo data does have hand pose. I would like to ask how to get the hand pose
> > > > 
> > > > 
> > > > We use [ProxyCap](https://zhangyux15.github.io/ProxyCapV2/) to estimate SMPL-X, but it is not open source yet. I recommend you to use [PyMAF-X](https://github.com/HongwenZhang/PyMAF/tree/smplx).
> > > 
> > > 
> > > thanks,I will try it
> > 
> > 
> > I know why the results are unclear, you need to set lpips_start_iter=10-30 (when to use lpips loss). Sorry
> > Yes, I also found this problem Also I found that the pose obtained with the [ReFit ](https://github.com/yufu-wang/ReFit)project includes hand information I compared the pose from the People Snapshot Dataset and found that the matrices are exactly the same If you train with the pose obtained from ReFit you might get better results.

sorry to disturb you, I  also want to refine the hand pose for the [PyMAF-X], can you share the pretrain model of the [partial_mesh](https://cloud.tsinghua.edu.cn/d/3bc20811a93b488b99a9) files .
Download the [pre-trained model](https://cloud.tsinghua.edu.cn/d/3bc20811a93b488b99a9)  for the author's provided links are not available. thanks very much
